### DISCUSSION TITLE
Keybind for suspending a process

### DISCUSSION THREAD

**Discussion by 825lol56** [winsiderss/systeminformer#1778] [Questions / Answers]

**Comment by Advait251206** [comment]
As far as I know, System Informer does not provide a built-in global or configurable keyboard shortcut specifically for **Suspend Process**.

The usual workflow is:

- Select the process.
- Right-click → **Suspend**.
- Or use the corresponding menu item under **Process**.

If you're looking for a faster workflow, a couple of alternatives are:

- Use the context menu keyboard shortcut (`Shift+F10` or the Menu key), then navigate to **Suspend**.
- Create a custom automation script (e.g., AutoHotkey) that interacts with System Informer and triggers the suspend command for the selected process.
- Use command-line tools such as `pssuspend` from Sysinternals if your workflow is heavily keyboard-driven.

I don't recall seeing a configurable hotkey system for individual process actions such as Suspend, Resume, or Terminate. If that's something you'd find useful, it may be worth opening a feature request, since process suspension is a common troubleshooting task and could benefit from a dedicated shortcut.
